### DISCUSSION TITLE
龙猫不是自己吧的吧主

### DISCUSSION THREAD

**Discussion by allMagicNB** [Meloong-Git/PCL#3356] [讨论]
https://tieba.baidu.com/f?kw=%E9%BE%99%E8%85%BE%E7%8C%AB%E8%B7%83&ie=utf-8

**Comment by Ad-closeNN** [comment]
[这什么](https://tieba.baidu.com/p/6794909132)，看着像记事本，但图标怎么是`PCL`的图标
~~太奇怪了~~
